src/controls/polyControl.ts

```diff
@@ -126,7 +132,7 @@ export function createPolyControls(
     idx++
   ) {
     controls[`p${idx}`] = new Control({
-      actionName: 'modifyPoly',
+      actionName: ACTION_NAME,
```

Comment:
Did we call it modifyPoly?
This is a terrible name.
Shouldn't be this just resizing? or modifying? it doesn't matter for this pr of course.

Reply:
I agree. BAD name. resizing... not percise. modifying  too general.

Reply:
i have to write an issue to cover the fact that actionName of wrapWithFireEvent will not be in sync with control.actionName.
Not sure if is worth fixing or just documenting
control.actionName is used for end transform event, while this one is used for in transform event.
The current situation must be at least explained somewhere

Reply:
I think it ought to be the same value. Why else?